src/commands/CommandHandler.ts

```diff
@@ -28,7 +28,8 @@ import { execRedactCommand } from "./RedactCommand";
 import { execImportCommand } from "./ImportCommand";
 import { execSetDefaultListCommand } from "./SetDefaultBanListCommand";
 import { execDeactivateCommand } from "./DeactivateCommand";
-import { execDisableProtection, execEnableProtection, execListProtections } from "./ProtectionsCommands";
+import { execDisableProtection, execEnableProtection, execListProtections, execConfigGetProtection,
+    execConfigSetProtection, execConfigAddProtection, execConfigRemoveProtection } from "./ProtectionsCommands";
```

Comment:
perhaps it would be better to just have one `execConfigChangeProtection` function that figures out what kind of change it is?

Reply:
I think it would be a bit nicer, but not a big deal.